src/components/member-profile/super-user-options.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import classNames from '@components/member-profile/member-profile.module.scss';
+import { userContext } from '../../store/user/user-context';
+
+const SuperUserOptions = ({ showSettings, isNoteworthy, TaskId }) => {
+  const {
+    isSuperUserMode,
+    showMemberTaskUpdateModal,
+    setshowMemberTaskUpdateModal,
+    setTaskId,
+    setIsnoteWorthy,
+  } = userContext();
+
+  const showModal = (e) => {
+    e.stopPropagation();
+    setshowMemberTaskUpdateModal(!showMemberTaskUpdateModal);

```

Comment:
Shouldn't we pass `true` . Currently this is more like a `toggleModal` than `showModal`
```suggestion
    setshowMemberTaskUpdateModal(true);
```

Reply:
yes correct, thanks for pointing out